### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by mugnivenko**

### CODE REVIEW

File: components/PayRuns/PayRunsHeader.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { useEffect, useState } from 'react';
 import { Button } from '../Button';
 import PayRunsFilters from './PayRunsFilters';
 import HeldPaymentsFilters from './HeldPaymentsFilters';
-import { checkEmptyFields } from '../../service/inputValidator'
+import { checkEmptyFields } from '../../service/formValidator'
```
Comment (akeamwoodsND) [inline components/PayRuns/PayRunsHeader.js:5]: service/formValidator

**Review by akeamwoodsND — COMMENTED** [review]
on going selection no longer works? (on http://localhost:3000/care-package)
Screenshot 2021-09-14 at 16 34 24

**Comment by mugnivenko** [comment]
ongoing date picker fixed

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Delete <div id='modal' /> from pages/index.js

**Pull request merged**